src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -45,6 +45,13 @@ export default (accessToken, userID, opts) => {
     followShowLoader: gravityLoader("follow_shows", {}, { method: "POST" }),
     unfollowShowLoader: gravityLoader("follow_shows", {}, { method: "DELETE" }),
     followedShowsLoader: gravityLoader("follow_shows", {}, { headers: true }),
+    followedShowLoader: trackedEntityLoaderFactory(
+      gravityLoader("follow_shows"),
+      "show_ids",
```

Comment:
Seeing this list of params (which can _definitely_ be confusing as to significance and ordering), makes me wish the factory took an options object.

``` typescript
type TrackedLoaderFactoryOptions {
  paramKey: string
  trackingKey: string
  entityKeyPath?: string
  entityIDKeyPath?: string
}
``` and so forth... but that's an improvement for another time!

Reply:
Yeah! It was getting a bit much with positional params. I think that'd be a good future improvement 👍

Reply:
Agreed